Pinta.Core/Classes/Palette.cs

```diff
@@ -151,13 +152,7 @@ public void Load (PaletteFormatManager paletteFormats, Gio.File file)
 			colors.TrimExcess ();
 			OnPaletteChanged ();
 		} catch (Exception e) {
-			var parent = PintaCore.Chrome.MainWindow;
-			PintaCore.Chrome.ShowUnsupportedFormatDialog (
-				parent,
-				PintaCore.PaletteFormats.Formats,
-				file.GetParseName (),
-				Translations.GetString ("Unsupported palette format"),
-				e.Message);
+			throw new PaletteLoadException (file.GetParseName (), e);
```

Comment:
Since the exception we're catching is one that we're in control of (from `LoadColors ()` below, maybe that method can directly throw a `PaletteLoadException` instead and avoid the `InnerException` stuff?